source/manage/cloud-data-export.rst

```diff
@@ -1,8 +1,138 @@
-Cloud Data Export
-=========================
+Mattermost Workspace Migration
+==============================
 
-If, after completing your 14-day free Cloud trial, you've decided to self-host your own Mattermost deployment, you may want to migrate your Cloud data to your self-hosted instance.
+This document outlines the process for migrating from Mattermost Cloud to a Mattermost self-hosted deployment. In the future, a process for migrating from Mattermost self-hosted to Mattermost Cloud will also be documented and provided here.
 
-We plan to publish documentation on how to perform a Cloud data export for channels, messages, users, and other bulk export data self-serve, with guidelines about running commands to export this data.
+Migrating between two installations is the same process regardless as to whether the source or destination of the migration is in the Cloud or self-hosted. **These steps will work for any Mattermost instance**, as long as the instances involved have a version of Mattermost later than 5.33.
```

Comment:
> Migrating between two installations is the same process regardless as to whether the source or destination of the migration is in the Cloud or self-hosted

It's not clear what is meant by `the same process` in this context?

Reply:
Thanks for catching this - it's from Ian's blog post so it references a bit that I took out.